### ISSUE TITLE
Outdated Base R support section

### ISSUE THREAD

**Issue by joshwlambert** [epiverse-trace/blueprints#116]
I believe the [table stating the versions of base R that are supported](https://epiverse-trace.github.io/blueprints/dependencies.html#base-r-support-schedule) is now outdated. We could consider dynamically rendering this from the GitHub action workflow, to try and prevent it being outdated in the future.

**Comment by joshwlambert** [comment]
Due to PR deploy previews only working on quarto documents without code we cannot use the same approach as the tidyverse blog post.

**Comment by joshwlambert** [comment]
> We could consider dynamically rendering this from the GitHub action workflow, to try and prevent it being outdated in the future.

It was also mentioned in #80 that automating this step might not be worth the effort.

**Comment by Bisaloo** [comment]
We don't know how future R versions will be numbered anyways so it's tricky to create the table for many years ahead. I believe the current situation, where it shows what the policy meant a couple of years ago and a couple of years ahead, is useful to understand the idea.

What do you think?

**Comment by joshwlambert** [comment]
Yes, I agree. This is likely more complication than is necessary. The only difference between the Epiverse table and tidyverse table is the number of past R releases.

Last point on the automation, we could use the {rversions} package, but it seems overly complex to make the query ourselves, e.g. https://github.com/r-hub/rversions/blob/master/R/rversions.R#L86-L118.

Therefore, I think we can rename the Current R version to R version (like the tidyverse table), which then removes the issue of the table becoming outdated and then close this issue without further action required.

@Bisaloo please let me know if you agree.

**Comment by Bisaloo** [comment]
Or "Current R version" -> "Latest R version" if you prefer?

Current makes sense to me as I read it as "Current R version as of year X".
